src/lib/deploy-actions.js

```diff
@@ -32,6 +32,14 @@ module.exports = async (config, isLocalDev = false, log = () => {}, filter = fal
         byBuiltActions: filter
       }
     }
+    if (inprocHook) {
```

Comment:
I see in the PR section `Motivation and Context`:
```
- aio app deploy [--action some-action]
  - for each action to be deployed fire an event with the config, which action is being deployed, isLocal
- aio app run [--local] 
  - for each action to be deployed
```

... that each action when deployed, the hook is fired. In my testing that is not the case, only one event is fired per deploy/run, not per action. What is the actual intent for the feature?

Reply:
Resolved in Slack chat: it's 1 event per deploy/run.